<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportSqlDialog</class>
 <widget class="QDialog" name="ExportSqlDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>497</width>
    <height>352</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export SQL...</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="2" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="labelTable">
       <property name="text">
        <string>Tab&amp;le(s)</string>
       </property>
       <property name="buddy">
        <cstring>listTables</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QListWidget" name="listTables">
       <property name="selectionMode">
        <enum>QAbstractItemView::MultiSelection</enum>
       </property>
       <property name="sortingEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetFixedSize</enum>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QPushButton" name="buttonSelectAll">
         <property name="text">
          <string>Select All</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonDeselectAll">
         <property name="text">
          <string>Deselect All</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>&amp;Options</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QCheckBox" name="checkColNames">
          <property name="text">
           <string>Keep column names in INSERT INTO</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="4" column="0">
         <widget class="QComboBox" name="comboOldSchema">
          <item>
           <property name="text">
            <string>Keep old schema (CREATE TABLE IF NOT EXISTS)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Overwrite old schema (DROP TABLE, then CREATE TABLE)</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="checkMultiple">
          <property name="text">
           <string>Multiple rows (VALUES) per INSERT statement</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QComboBox" name="comboWhat">
          <item>
           <property name="text">
            <string>Export everything</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Export schema only</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Export data only</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="checkOriginal">
          <property name="text">
           <string>Keep original CREATE statements</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>listTables</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ExportSqlDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>258</x>
     <y>331</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonDeselectAll</sender>
   <signal>clicked()</signal>
   <receiver>ExportSqlDialog</receiver>
   <slot>doDeselectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>488</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonSelectAll</sender>
   <signal>clicked()</signal>
   <receiver>ExportSqlDialog</receiver>
   <slot>doSelectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>140</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ExportSqlDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>317</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboWhat</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>ExportSqlDialog</receiver>
   <slot>whatChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>234</y>
    </hint>
    <hint type="destinationlabel">
     <x>492</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>showCustomCharEdits()</slot>
  <slot>whatChanged(int)</slot>
 </slots>
</ui>
